pandas/tests/tools/test_to_datetime.py

```diff
@@ -2552,7 +2688,12 @@ def test_day_not_in_month_raise(self, cache):
 
     @pytest.mark.parametrize("arg", ["2015-02-29", "2015-02-32", "2015-04-31"])
     def test_day_not_in_month_raise_value(self, cache, arg):
-        msg = f'time data "{arg}" at position 0 doesn\'t match format "%Y-%m-%d"'
+        msg = "|".join(
+            [
+                "^day is out of range for month$",
```

Comment:
this one doesn't show the position - I've opened a separate PR to address this, https://github.com/pandas-dev/pandas/pull/50464

I don't think it needs to be considered a blocker